Makefile

```diff
@@ -32,9 +32,9 @@ SPEC_FETCHED_PATCHES=patches/spec.fetched.json
 ##
 # openapi-generator-cli - docker
 ##
-OPENAPI_CODEGEN_TAG=latest
+OPENAPI_CODEGEN_SHA=sha256:be8c2ef6be22f695410c2cc13d0ec7fdf2533fc88a7f17288ad758b7679de8df
 
-OPENAPI_CODEGEN_IMAGE=openapitools/openapi-generator-cli:${OPENAPI_CODEGEN_TAG}
+OPENAPI_CODEGEN_IMAGE=openapitools/openapi-generator-cli@${OPENAPI_CODEGEN_SHA}
```

Comment:
*NOTE:* the `openapitools/openapi-generator-cli:latest` tag includes a breaking change for Java (it generated uncompilable code).  This locks the image to the last-known-good SHA corresponding to the `latest` tag in order to fix the build without introducing unexpected changes.  Ideally we would replace this SHA with an immutable version tag (e.g., `v6.3.1` instead of `latest`) when we're ready to do so.